project/views/report.py

```diff
@@ -616,7 +616,7 @@ def get_context_data(self, **kwargs):
                 "pop_chart": pop_chart,
                 "pop_table": pop_chart.get_series(),
                 "conso_pop_chart": conso_pop_chart,
-                "conso_pop_table": add_total_line_column(conso_pop_chart.get_series(), line=False),
+                "conso_pop_table": add_total_line_column(conso_pop_chart.get_series(), line=False, column=False),
```

Comment:
Si `line` et `column` sont nuls, la table est retournée telle qu'elle est passée dans la fonction
```suggestion
                "conso_pop_table": conso_pop_chart.get_series(),
```

Reply:
Je sais : ) mais va savoir pourquoi ça bug si tu fais ça. C'est les graphiques INSEE qu'on a masqués et qu'on doit refondre, du coup je ne me suis pas attardé dessus. Je te propose de voir ça dans la prochaine itération.